### ISSUE TITLE
[Installation question] The mod-release comes without weidu.exe?

### ISSUE THREAD

**Issue by rubyFeedback** [Gibberlings3/Grey_The_Dog#20]
This is mostly a question. When trying to install the mod, there was no .exe, e. g. grey.exe or something. I assume we can just use any .exe that is weidu and rename it, but since most other mods come with .exe I was not sure. Do you think the .exe could be added to the downloads directly?

For instance when downloading bridge block from lava:

https://downloads.weaselmods.net/download/bridges-block/

I get a similar directory structure, but at the base directory also:

   setup-BridgesBlock.exe

Then just clicking on it is quite convenient IMO. At any rate, thank you
for reading this issue request.

**Comment by Gitjas** [comment]
Always download the attached zip from the latest release. There should be an exe inside. 
https://github.com/Gibberlings3/Grey_The_Dog/releases

**Comment by Argent77** [comment]
I have to agree with ruby. The zip package of the latest release ([v10](https://github.com/Gibberlings3/Grey_The_Dog/releases/tag/v10)) does not contain any setup binaries. It includes only the macOS-specific .command file. The actual setup binaries for all supported platforms are missing.

This is also noticeable by the download size which is basically identical to the .iemod package size:
<details>
<summary>Screenshot</summary>
<img src="https://github.com/user-attachments/assets/775095e6-43c3-4630-b12d-3f9a03675172" alt="Screenshot">
</details>

**Comment by Gitjas** [comment]
I finally noticed now, thanks! I'm in touch with AL|EN who's autopacker I am using for creating the attached zip.
